cmd/eskip/eskip.go

```diff
@@ -113,6 +113,18 @@ func main() {
 		exitHint(err)
 	}
 
+	in, out, err = addDefaultMedia(cmd, in, out)
+
+	if err != nil {
+		exitHint(err)
+	}
+
+	writeClient, err := createWriteClient(out)
```

Comment:
cool, but still why not to handle both the createWriteClient and the createReadClient the same way? either both could be here or else.